src/modules/bigquery.js

```diff
@@ -0,0 +1,117 @@
+import axios from 'axios';
+import _ from 'lodash';
+
+export default class BigQuery {
+  constructor(pageTokenMap) {
+    this.pageTokenMap = pageTokenMap;
+  }
+
+  pageData = (query, queryResults, columns, token) =>
+    new Promise(resolve => {
+      let rawData = {};
+      const data = [];
+
+      const { jobId } = queryResults.jobReference;
+      const { projectId } = queryResults.jobReference;
```

Comment:
You can say `const { jobId, projectId } = queryResults.jobReference;` to unpack both variables in one line